js/app.js

```diff
@@ -0,0 +1,107 @@
+console.log('yo')
```

Comment:
in the directions it asked you to paste which questions you were answering as comments above the answers, so i had to do some guesswork to figure out which questions you were trying to answer

also the code generates errors -- did you check it in your browser as you were going?????